### PR TITLE
Latest rover changes

### PR DESCRIPTION
**Pull request by mstuettgen**
Renamed the files to match the models name and changed the PWM_MIN PWM_MAX values

### CODE REVIEW

File: src/examples/rover_steering_control/main.cpp [hunk 1]
```diff
@@ -156,6 +156,23 @@ void control_attitude(const struct vehicle_attitude_setpoint_s *att_sp, const st
 		      struct actuator_controls_s *actuators)
 {
 
+  //double r,p,y,t;
+  
+
+  
+  //printf("vehicle_attitude_setpoint:\n");
+  //print values only when they change to get less spam in console
+  if((double)att_sp->roll_body > 0.0) { printf("att_sp->roll_body: %8.4f\n" , (double)att_sp->roll_body); }
```
Comment (LorenzMeier) [inline src/examples/rover_steering_control/main.cpp]: This code part has issues with types and with the range check - you have to check +- here. Plus we should not have commented out code in master or controllers printing to the console. Either guard it with a verbose or debug flag or please remove it entirely.

**Comment by LorenzMeier** [comment]
@mstuettgen It would be great if you could address my review feedback, I'd be happy to merge it then.

**Comment by mstuettgen** [comment]
Hi Lorenz, I just returned from a business trip to south africa,  i will take a look in the next das. 
CheersMarcel


Von meinem Samsung Gerät gesendet.

-------- Ursprüngliche Nachricht --------
Von: Lorenz Meier <notifications@github.com> 
Datum: 15.03.2015  5:46 PM  (GMT+01:00) 
An: PX4/Firmware <Firmware@noreply.github.com> 
Cc: Marcel Stüttgen <m@rcel-stuettgen.de> 
Betreff: Re: [Firmware] Latest rover changes (#1875) 

@mstuettgen It would be great if you could address my review feedback, I'd be happy to merge it then.

**Comment by mstuettgen** [comment]
Hi Lorenz, I finally have time to look at the changes, what is the standard debug / verbose flag you use in the pixhawk firmware?

are you using something like "#ifdef _DEBUG" ?

**Comment by LorenzMeier** [comment]
You need to define your own command line argument.

**Comment by mstuettgen** [comment]
Hi lorenz, i removed the debug code because I dont have much time at the moment. Should not be a problem because we are not using the app at the moment anyways.

I have finished the prototype of the rover and we will start fine-tuning everything, there will be another PR coming soon !

**Pull request merged**